x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots/observable.ts

```diff
@@ -42,7 +45,7 @@ export function screenshotsObservableFactory(
               mergeMap(() => openUrl(captureConfig, driver, url, conditionalHeaders, logger)),
               mergeMap(() => getNumberOfItems(captureConfig, driver, layout, logger)),
               mergeMap(async itemsCount => {
-                const viewport = layout.getViewport(itemsCount);
+                const viewport = layout.getViewport(itemsCount) || getDefaultViewPort();
```

Comment:
heads up @poffdeluxe, these changes might conflict in the branch you're working on regarding re-using the page object for multiple urls